controllers/alertmanagerconfig_controller.go

```diff
@@ -257,7 +257,7 @@ func (r *AlertmanagerConfigReconciler) deleteWebhooksFromRelatedAlerts(ctx conte
 	}
 
 	for _, alert := range alerts.Items {
-		alert.Spec.NotificationGroups = []coralogixv1alpha1.NotificationGroup{{}}
+		alert.Spec.NotificationGroups = nil
```

Comment:
In this case it resets (and then updates) the notification groups.  the original code just results in validation issues since an empty list of notificationgroups is not allowed somehow (but NIL is)

Reply:
correct, thanks.